openvino_xai/methods/white_box/torch.py

```diff
@@ -209,9 +262,24 @@ def __init__(
         normalize: bool = True,
         **kwargs,
     ) -> None:
-        super().__init__(*args, **kwargs)
         self._use_gaussian = use_gaussian
         self._use_cls_token = use_cls_token
+        super().__init__(*args, **kwargs)
+
+    def _find_feature_module(self, module: torch.nn.Module) -> torch.nn.Module:
+        """Detect feature module in the model."""
+        # Find the 3rd last LayerNorm module
```

Comment:
```suggestion
        """Detect feature module in the model by finding the 3rd last LayerNorm module."""
```

Reply:
Done in https://github.com/openvinotoolkit/openvino_xai/pull/64/commits/6617ccbf73b700f4cd60646ed49d46673e16060d